.travis.yml

```diff
@@ -4,5 +4,6 @@ jdk:
   - openjdk11
   - oraclejdk8
 install: /bin/true
-script: mvn install --quiet -Dgpg.skip=true -DskipTests=true
-
+script: mvn install
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Since this is generally an antipattern, could you add a comment with a link to codecov's docs explaining this?

https://docs.codecov.io/docs/about-the-codecov-bash-uploader

Reply:
Sure!